components/list/list-item-drag-handle.js

```diff
@@ -169,13 +175,23 @@ class ListItemDragHandle extends LitElement {
 		}
 		this._dispatchAction(action);
 		e.preventDefault();
-	}
-
-	_onBlur() {
-		this._keyboardActive = false;
-		if (!this._tabbing) {
-			this._dispatchAction(dragActions.save);
+		e.stopPropagation();
+		const cell = findComposedAncestor(this, (parent) =>  parent.hasAttribute && parent.hasAttribute('draggable'));
```

Comment:
Is this the demo component that holds the `d2l-list`? This is the ancestor of the list that only has this attribute for demo purposes; is it possible to wait for the list component's update instead?

Reply:
It is looking for `d2l-list-item`